gooddata-sdk/gooddata_sdk/catalog/data_source/declarative_model/data_source.py

```diff
@@ -80,18 +104,23 @@ def __init__(
         self.permissions = permissions if permissions is not None else []
 
     @classmethod
-    def from_api(cls, entity: dict[str, Any]) -> CatalogDeclarativeDataSource:
+    def from_api(cls, entity: dict[str, Any], include_nested_structures: bool = True) -> CatalogDeclarativeDataSource:
```

Comment:
Could you please clarify the purpose of include_nested_structures in from_api method? I understand the purpose in to_api method, but I do not see any in from_api method.

Reply:
removed, was leftover